scripts/haas_network.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=HaaS Network Server 
+After=network.target
```

Comment:
I tried testing dependencies using httpd.
rather than ``after=`` I used ``requires=`` 
what it does is that if you start haas_network.service and it sees that it contains a directive called
``requries=httpd.service`` 
it will check whether the service is running or not. If not it starts the service before it starts this service.

My thinking was that we should have dependencies on both httpd, and database for now since all three components are running on the same server and change it when we actually come to the point of running them on different machines. 

The reason being simply that dependencies makes for a self-check style of running services which in my opinion is more robust and less maintenance prone. 

But I am open to suggestions. 
@henn @gsilvis any thoughts, besides just two of us discussing here. :)